server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -664,7 +667,11 @@ public void update(LiveTServerSet current, Set<TServerInstance> deleted,
           default:
             log.trace("\'none - no action\' or invalid value provided: {}", action);
         }
-        log.debug("gc post action {} complete", action);
+
+        long actionComplete = System.currentTimeMillis();
```

Comment:
Best to use nanos when computing time durations within a process, since it's not dependent on system clock changes.